.github/DEVELOPMENT.md

```diff
@@ -32,20 +33,42 @@ Install the .NET MAUI workload using the versions from a particular branch:
 
 For example, the "preview.14" branch:
 
+
+Windows:
+
 ```
 iwr https://aka.ms/dotnet/maui/maui-install.ps1 -OutFile maui-install.ps1;
 .\maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
-``` 
+```
+
+MacOS:
+
+```
+curl -O https://aka.ms/dotnet/maui/maui-install.ps1
+chmod +x maui-install.ps1
+.\maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
```

Comment:
This script is still failing due to a missing sudo on line 116 where we install workloads. Could we potentially add the following to line 116 of the script (not available on github as far as I know).

```
    if ($IsLinux -or $IsMacOS) {
        sudo dotnet workload install @Workloads --from-rollback-file $rollbackFile --configfile $nugetConfigFile
    } else {
        dotnet workload install @Workloads --from-rollback-file $rollbackFile --configfile $nugetConfigFile
    }
```